e2e/testinfra/terraform/modules/service_account/service_account.tf

```diff
@@ -40,3 +41,16 @@ resource "google_project_iam_member" "gcp_sa_role" {
   member  = "serviceAccount:${google_service_account.gcp_sa.email}"
   project = data.google_project.project.id
 }
+
+// Create IAM bindings directly to the KSA(s) for BYOID
+resource "google_project_iam_member" "k8s_sa_role_rootsync" {
+  role    = var.role
+  member  = "serviceAccount:${data.google_project.project.project_id}.svc.id.goog[config-management-system/root-reconciler]"
+  project = data.google_project.project.id
+}
+
+resource "google_project_iam_member" "k8s_sa_role_reposync" {
```

Comment:
This isn't used right now. We can add it when we add test coverage for RepoSync with k8sserviceaccount.